app/views/devise/registrations/new.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= f.label :name %><br />
     <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
   </div>
-
+  
```

Comment:
Please avoid changes like this, which are just whitespace.

Reply:
Whitespace removed